output/cloud/v1/output.go

```diff
@@ -472,4 +533,42 @@ func (out *Output) pushMetrics() {
 	}).Debug("Pushing metrics to cloud finished")
 }
 
+func (out *Output) runFlushRequestMetadatas() {
+	t := time.NewTicker(out.config.TracesPushInterval.TimeDuration())
+
+	for i := int64(0); i < out.config.TracesPushConcurrency.Int64; i++ {
+		out.outputDone.Add(1)
+		go func() {
+			defer out.outputDone.Done()
+			defer t.Stop()
+
+			for {
+				select {
+				case <-out.stopSendingMetrics:
```

Comment:
It seems like this is related more to the metrics, not the traces. Out of curiosity, why is this here? :thinking: 

I'm also asking since generally code of the `runFlushRequestMetadatas` isn't so different from the v2 and probably could be extracted to the insights' package

Reply:
Our backend does not support aborting test runs. I think it makes sense to stop sending data to our backend as well once the core backend figures out that the test `shouldStopSendingMetrics(...)`.

Reply:
A semantic difference exists between `<-out.stopSendingMetrics` and `<-out.stopOutput`. Stop output is for a graceful exit - we still flush what's remaining. In case of "stop sending metrics", we exit and stop sending data. We are not coupling two domains here. The tracing metrics are highly correlated with HTTP metrics. There is no reason to send data to the insights backend if we don't send data to the metrics backend.